tests/unit/options.js

```diff
@@ -1676,17 +1676,35 @@ exports.nonbsp = function (test) {
 /** Option `nocomma` disallows the use of comma operator. */
 exports.nocomma = function (test) {
   // By default allow comma operator
-  TestRun(test)
+  TestRun(test, "nocomma off by default")
     .test("return 2, 5;", {});
```

Comment:
I've been thinking about the same question, but had dismissed it as "it's an early error anyway". Do you think we should be more aggressive there?